packages/create-llama/helpers/run-app.ts

```diff
@@ -64,14 +78,11 @@ export async function runApp(
     });
   } else {
     return new Promise((resolve, reject) => {
-      backendAppProcess = spawn(backendCommand, backendArgs, {
+      backendAppProcess = createProcess(backendCommand, backendArgs, {
         stdio: "inherit",
-        cwd: appPath,
+        cwd: path.join(appPath, "backend"),
```

Comment:
@thucpn @leehuwuj this looks wrong as this is the case where the app is just in one folder